x-pack/plugin/esql/qa/testFixtures/src/main/resources/lookup-join.csv-spec

```diff
@@ -1252,7 +1252,7 @@ ignoreOrder:true
 
 lookupIndexInFromRepeatedRowBug
 // Test for https://github.com/elastic/elasticsearch/issues/118852
-required_capability: join_lookup_v11
```

Comment:
Oof, that's a lot of bumps; I think we're at the point where we should rather add new capabilities rather than bumping, as that's less cumbersome. (It's fine to do, of course, and this is not criticizing the PR.)